python/mdvtools/charts/heatmap_plot.py

```
@@ -31,4 +31,7 @@ def set_x_axis(self, axis_labels, axis_title):
     def set_y_axis(self, axis_labels, axis_title):
         self.plot_data["y_axis"] = {"labels": axis_labels, "title": axis_title}
 
+    def set_axis(self, xtextSize, ytextSize, xsize, ysize, xtickfont, ytickfont):
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)